monitors.yml

```diff
@@ -183,3 +183,20 @@ monitors:
         warning: 400000000
     query: avg:system.mem.free{*} by {host}
     type: metric alert
+
+  - message: |
+      {{#is_alert}}
+      zuul process wait time is very high
+      {{/is_alert}}
+
+      @zxkuqyb@gmail.com
+      @slack-pirateshiprevenge
+    multi: false
+    name: zuul process wait time
+    options:
+      notify_no_data: false
+      require_full_window: false
+      thresholds:
+        critical: 100000
+    query: avg:zuul.pipeline.check_github.job.bonnyci_run_check.wait_time.max{host:zuul}
```

Comment:
I think you have to apply average at some point so that datadog can turn X values into 1 that it can compare its threshold check against. I'm not sure what the default length of that period is, one doesn't seem to be defined.

There is such a thing as outlier monitoring, so that if something here change outside of some pre-defined norm then you can be alerted about that which would mean you don't have to guess at a threshold. I don't know how hard they are to set up. 

My nits would be there's no warning level just a critical by which point things are bad, is that all we care about? Also that we're using the run_check in the check_github pipeline for testing here. Whilst this will be the most common run job it means we only know about failures here, is there a way we can know about a global wait time? does that even make sense. 

I'm happy to see this merge because it's better than nothing and we can tweak as we learn more.